ci/docker/Dockerfile

```diff
@@ -36,11 +36,15 @@ RUN apk --update add --no-cache ca-certificates curl openssl binutils xz \
 COPY --from=0 /build/jre /opt/mongoose
 COPY --from=0 /build/libs/mongoose-*.jar /opt/mongoose/
 ADD ci/docker/entrypoint.sh /opt/mongoose/entrypoint.sh
+ADD ci/docker/entrypoint_debug.sh /opt/mongoose/entrypoint_debug.sh
+
+RUN apk --no-cache add openjdk11 --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
```

Comment:
This will increase the production image size. Let's have separate debug dockerfile and image

Reply:
Do you mean separate images?

Reply:
exactly